src/extension.ts

```diff
@@ -49,6 +49,7 @@ function selectorsCreator(wf: WorkspaceFolder) {
                 ({
                     language: l,
                     pattern: relativePattern,
+                    scheme: 'file'
```

Comment:
You know when doing code reviews that the following situation happens more than often: When there are very few changes, like in this PR, then the code review is done with high precision and you will get more comments than that you actually wrote lines of code.

So to fix this I am going to write this useless comment to prove that my above statement is true and that I can comment way too much information for a little line change, which looks perfect by the way!

Also, when a PR is has many many files changed and is fairly big then the code reviews sometimes look like:

> LGTM

But not this one, this PR is small and thus needs a long comment. Okay okay, I frankly don't know what I am doing at this point but I hope you are enjoying your day today. 🎉